media/js/firefox/new/de-experiment.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment_firefox_new_de_hearts',
+        variations: {
+            'v=a': 10,
+            'v=b': 10
+        }
+    });
+
+    cop.init();
+
+    /* update dataLayer with experiment info */
+    var href = window.location.href;
+    if(href.indexOf('v=a') !== -1) {
+        window.dataLayer.push({
+            'data-ex-variant': 'de-page',
+            'data-ex-name': 'Berlin-Campaign-Landing-Page'
+        });
+    } else if (href.indexOf('v=b') !== -1) {
+        window.dataLayer.push({
+            'data-ex-variant': 'campaign-page',
+            'data-ex-name': 'Berlin-Campaign-Landing-Page'
+        });
+    }
```

Comment:
It's enough to do the analysis the hard way but it sounds like this will help us do it more easily. (Maybe just in GA without needing DataStudio?)

Reply:
I don't think the callback is a good way to do this because I don't want the dataLayer updated before the redirect happens. But maybe I am misunderstanding how it works?

Reply:
I added this to the firefox_new_de_b bundle for now.